### PR TITLE
Updated go and refs versions, added radix-batch-scheduler to white-list

### PR DESCRIPTION
**Pull request by satr**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ RUN golint `go list ./cmd/...` && \
 # build
 RUN CGO_ENABLED=0 GOOS=linux go build -ldflags "-s -w" -a -installsuffix cgo -o /usr/local/bin/radix-acr-cleanup ./cmd/acr-cleanup/.
 
-FROM mcr.microsoft.com/azure-cli:2.2.0
+FROM golang:1.17.6-alpine3.15
```
Comment (nilsgstrabo) [inline Dockerfile:28]: We must use the `mcr.microsoft.com/azure-cli` image to be able to exexute `az cli` commands used in acr.go.
Version 2.2.0 is a bit old, so it should be updated to a newer version. Latest is 2.9.1

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM golang:alpine3.11 as builder
+FROM golang:1.17.6-alpine3.15 as builder
```
Comment (nilsgstrabo) [inline Dockerfile:1]: I updated this to `golang:1.17-alpine` in my update go modules PR a few days ago.

File: Dockerfile (hunk 1 above)
Comment (satr) [reply Dockerfile:28]: Fixed

File: Dockerfile (hunk 2 above)
Comment (satr) [reply Dockerfile:1]: I think I've seen it, but I did not find it. Fixed

**Review by nilsgstrabo — APPROVED** [review]

**Pull request merged**